api/STYLE.md

```diff
@@ -85,7 +85,7 @@ In addition, the following conventions should be followed:
   at the top that specifies the next free field number. E.g.,
 
   ```
-  // [#comment:next free field: 28]
+  // [#next-free-field: 28]
```

Comment:
Can you update the above comment that this field will be automatically generated and/or just remove this entirely?